assets/js/atomic/blocks/product/category-list/style.scss

```
@@ -0,0 +1,23 @@
+.wc-block-layout .wc-block-components-product-category-list {
+	margin-top: 0;
+	margin-bottom: $gap-small;
```

And make this margin relative too:
```suggestion
	margin-bottom: rem($gap-small);
```